### PR TITLE
TKO-79 BE: Workshop creation 

### PR DESCRIPTION
**Pull request by MikkalMc**
# Description
Created the general workflow for creating workshops based on the mockup. Added migrations, models, service, controller and route: /api/workshop/create

### CODE REVIEW

File: database/migrations/2019_04_30_183052_workshop_attendence.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?php
```
Comment (luisjg) [inline database/migrations/2019_04_30_183052_workshop_attendence.php]: Seems to be an empty migration. This doesn't spark joy

**Review by luisjg — COMMENTED** [review]
This looks good but please see comment.

File: database/migrations/2019_04_30_183052_workshop_attendence.php (hunk 1 above)
Comment (MikkalMc) [reply database/migrations/2019_04_30_183052_workshop_attendence.php]: oh oops didnt mean to commit dat boi

**Review by luisjg — APPROVED** [review]
👍

**Pull request merged**